### DISCUSSION TITLE
Contact Form

### DISCUSSION THREAD

**Discussion by Fahim-Ferdous** [adityatelange/hugo-PaperMod#564] [Q&A] [answered]
Hey there, really liked your theme but it lacks a contact form. Is it possible to add one? I don't know CSS or anything about UI/UX, so my form gets pretty messy...

**Comment by pixelfabs** [comment, marked as answer]
a contact form is very easy to add, just follow these steps:

1. make a new page, say `form.md` in `/content`
2. add frontmatter and code
```
---
title: Form
---
{{< rawhtml >}}
<form action="/action_page.php">
  <label for="fname">First name:</label><br>
  <input type="text" id="fname" name="fname" value="John"><br>
  <label for="lname">Last name:</label><br>
  <input type="text" id="lname" name="lname" value="Doe"><br><br>
  <input type="submit" value="Submit">
</form> 
{{< /rawhtml >}}
```
3. add following code to /assets/css/extended/blank.css

```
input[type=text], select {
  width: 100%;
  padding: 12px 20px;
  margin: 8px 0;
  display: inline-block;
  border: 1px solid #ccc;
  border-radius: 4px;
  box-sizing: border-box;
}

input[type=submit] {
  width: 100%;
  background-color: #4CAF50;
  color: white;
  padding: 14px 20px;
  margin: 8px 0;
  border: none;
  border-radius: 4px;
  cursor: pointer;
}

input[type=submit]:hover {
  background-color: #45a049;
}
```

4. Save & Enjoy

The form elements blend in when using the default CSS. It then looks like the fields are missing. You can take any form CSS you like and add it to /assets/css/extended/blank.css

**Reply by abishek** [reply]
when I add a page like this, I am not able to access it!? In this example, the form is accessible at /form/ right?
